### PR TITLE
Update Info.js

### PR DESCRIPTION
**Pull request by eddie7788**

### CODE REVIEW

**Review by LvandoApps — CHANGES_REQUESTED** [review]
do ensure that tests pass in the feature branch before merging into dev as dev is intended for functional releases, in this case theres a syntax error in the line you added

File: frontend/src/components/Sidepanel/Panels/Info/Info.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ const Info = () => {
             <th>{getStateName(state.id)}</th>
             {alphabet.map(symbol => <td key={symbol}>
               {Object.entries(transitionMap)
-                .filter(([key]) => key.split(',')[0] === state.id && key.split(',')[1] === symbol)
+                .filter(([key]) => key.split(',')[0] === state.id toString() && key.split(',')[1] === symbol)
```
Comment (LvandoApps) [inline frontend/src/components/Sidepanel/Panels/Info/Info.js:77]: .filter(([key]) => key.split(',')[0] === state.id.toString() && key.split(',')[1] === symbol)

this should fix the syntax error, and then ensure linting tests pass even outside this

**Comment by ire4ever1190** [comment]
Also try and follow the [git conventions](https://github.com/automatarium/automatarium/blob/dev/CONTRIBUTING.md#git-conventions) in future. Not a problem for this PR, but good to keep in mind going forward

File: frontend/src/components/Sidepanel/Panels/Info/Info.js (hunk 1 above)
Comment (eddie7788) [reply frontend/src/components/Sidepanel/Panels/Info/Info.js:77]: thank you i will try again.

**Review by ire4ever1190 — APPROVED** [review]

**Review by LvandoApps — APPROVED** [review]
all good now, good job!

**Pull request merged**